src/main/java/io/vertx/core/http/impl/Http1xServerConnection.java

```diff
@@ -21,14 +21,7 @@
 import io.netty.channel.FileRegion;
 import io.netty.handler.codec.DecoderResult;
 import io.netty.handler.codec.TooLongFrameException;
-import io.netty.handler.codec.http.DefaultFullHttpResponse;
-import io.netty.handler.codec.http.HttpContent;
-import io.netty.handler.codec.http.HttpObject;
-import io.netty.handler.codec.http.HttpRequest;
-import io.netty.handler.codec.http.HttpResponseStatus;
-import io.netty.handler.codec.http.HttpUtil;
-import io.netty.handler.codec.http.HttpVersion;
-import io.netty.handler.codec.http.LastHttpContent;
+import io.netty.handler.codec.http.*;
```

Comment:
Can we update the [code style guidelines](https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines) to settle on on thresholds for wildcards usage in imports and static imports?

I'm not saying there's an issue with **your** setup. Just annoying to have a lot of changes like this in pull requests.

Reply:
let's not be that much regarding :-) , it's my default setup - I agree that those should be left out when submitting a PR and/or handled separately